loopd/config.go

```diff
@@ -28,7 +28,7 @@ type lndConfig struct {
 
 type viewParameters struct{}
 
-type config struct {
+type Config struct {
 	ShowVersion    bool   `short:"V" long:"version" description:"Display version information and exit"`
```

Comment:
Looking at this now I am wondering whether this isn't a layering violation. We have a library that also defines command line options. It probably is a practical choice to prevent duplication the struct in the cli.

Reply:
The command line library is very nice that it allows config structs to easily be hierarchical. This is a big part of what makes the Grub possible. But short command line options must be unique because they only consist of one letter and cannot be hierarchical. I don't think this is a big issue. I don't like the short versions anyway because I can never remember which tool uses which (is `-v` version or verbose? is it lowercase or uppercase? and so on).

Reply:
I just mean that ideally the names of command lines flags don't belong in a library. Someone else may use a different command line parsing library and then those don't work anyway.